src/webnn_native/GraphBuilder.cpp

```diff
@@ -89,6 +94,19 @@ namespace webnn_native {
     OperandBase* GraphBuilderBase::Conv2d(OperandBase* input,
                                           OperandBase* filter,
                                           Conv2dOptions const* options) {
+        if (options != nullptr && options->activation != nullptr) {
+            auto operatorType = options->activation->GetOperatorType();
+            if (operatorType == OperatorType::Clamp) {
```

Comment:
Should we handle this in each backend? What should we do if one backend could fuse clamp?

Reply:
For OpenVINO backend which can fuse clamp: due to ClampOperator has not been added into graph like operands, it's inputs(min/max) can't be found in graph. So:
1. If clamp's min/max are constants, we can convert constants to float values. In this case, we can add clamp node in backend implementation.
2. If clamp's min/max are some other nodes' outputs (not constants), we can not add clamp node in backend implementation because we can't find its inputs in graph.

So we add clamp node in GraphBuilder.cpp as a workaround now. As discussed with @fujunwei , we may need to add derived class from GraphBuilderBase for different backends.

Reply:
I am fine with this workaround given OpenVINO can fuse clamp by its graph compiler and DML doesn't support fuse clamp today. My suggestion would be:
1. document this workaround in the comment for conv2d and batchNorm
2. open an issue that track this workaround, so we can revisit it once a backend requires fusing clamp.